test-projects/basic/.env.example

```diff
@@ -2,3 +2,4 @@
 # CLOUDINARY_KEY=abc123
 # CLOUDINARY_SECRET=abc123
 PORT=1
+MONGODB_URI=mongodb://127.0.0.1:27017/tp_basic
```

Comment:
No need to add the value here, we don't want to force its usage, instead we can rely on the fallback.

Reply:
I think providing this here provides a good example for users to follow?

Reply:
`dotenv-safe` will read `.env.example` for which env vars are _required_. With this change, if the environment doesn't provide a `MONGODB_URI`, `dotenv-safe` will throw an error, which is not what we want (we want it to default to the localhost fallback when not set), especially as this is a test project and we will be using an in-memory mongo server which is passed in manually (ie; not via an env var)

Reply:
Rebased and removed.